packages/ember-routing/lib/index.d.ts

```diff
@@ -192,11 +192,12 @@ export const Route: {
   teardownViews(): void;
   isRouteFactory: boolean;
 };
+
 export const QueryParams: {
```

Comment:
I think this needs to get turned into a `class`, TypeScript is complaining.

Reply:
ohoh, someone has angered TypeScript

Reply:
😂 😂